pkg/common/config/config.go

```diff
@@ -337,6 +341,31 @@ func GetCnsconfig(ctx context.Context, cfgPath string) (*Config, error) {
 	return cfg, nil
 }
 
+// GetFeatureStatesConfig returns feature states config from specified file path
+func GetFeatureStatesConfig(ctx context.Context, featureStatesCfgPath string, cfg *Config) error {
+	log := logger.GetLogger(ctx)
+	log.Debugf("GetFeatureStatesConfig called with featureStatesCfgPath: %s", featureStatesCfgPath)
+	//Fetch feature state information in the csi-feature-states.conf if it exists
```

Comment:
Are we using a common path for feature states across CNS-CSI and PVCSI?

Reply:
no, we have different paths. Vanilla and GC uses same path as default.

```
// DefaultSVFeatureStateConfigPath is the default path of csi feature states config file in Supervisor Cluster
DefaultSVFeatureStateConfigPath = "/etc/vmware/wcp/csi-feature-states/csi-feature-states.conf"

// DefaultGCFeatureStateConfigPath is the default path of csi feature states config file in Guest Cluster
DefaultGCFeatureStateConfigPath = "/etc/cloud/csi-feature-states/csi-feature-states.conf"

// DefaultVanillaFeatureStateConfigPath is the default path of csi feature states config file in Vanilla Cluster
DefaultVanillaFeatureStateConfigPath = "/etc/cloud/csi-feature-states/csi-feature-states.conf"
```